developer-setup.md

```diff
@@ -0,0 +1,174 @@
+
+# Development Workflow
+
+## Prerequisites
+
+* You have Go 1.14 installed on your local host/development machine.
+* You have Docker installed on your local host/development machine. Docker is required for building jiva container images and to push them into a Kubernetes cluster for testing.
+* You have `kubectl` installed. For running integration tests, you will require an existing single node cluster with [openebs](https://blog.openebs.io/how-to-install-openebs-with-kubernetes-using-minikube-2ed488dff1c2) installed. Don't worry if you don't have access to the Kubernetes cluster, raising a PR with the jiva repository will run integration tests for your changes against a Minikube cluster.
+
+## Initial Setup
+
+### Fork in the cloud
+
+1. Visit https://github.com/openebs/jiva
+2. Click `Fork` button (top right) to establish a cloud-based fork.
+
+### Clone fork to local host
+
+Place openebs/jiva's code on your `GOPATH` using the following cloning procedure.
+Create your clone:
+
+```
+mkdir -p $GOPATH/src/github.com/openebs
+cd $GOPATH/src/github.com/openebs
+
+# Note: Here user= your github profile name
+git clone https://github.com/$user/jiva.git
+
+# Configure remote upstream
+cd $GOPATH/src/github.com/openebs/jiva
+git remote add upstream https://github.com/openebs/jiva.git
+
+# Never push to upstream master
+git remote set-url --push upstream no_push
+
+# Confirm that your remotes make sense:
+git remote -v
+```
+> **Note:** If your `GOPATH` has more than one (`:` separated) paths in it, then you should use *one of your go path* instead of `$GOPATH` in the commands mentioned here. This statement holds throughout this document.
+
+### Building and Testing your changes
+
+* To build the jiva binary
+```
+go build
+```
+
+* To build the docker image
+```
+make build
+```
+
+* Test your changes
+Integration tests are written in shell script and jiva controller and replicas are run as docker containers.
+To run the run the integration tests locally, run
```

Comment:
nit: `run the` is repeated in the statement.

Reply:
addressed